javascript/statements.json

```diff
@@ -964,6 +964,61 @@
           }
         }
       },
+      "for_await_of": {
+        "__compat": {
+          "description": "<code>for await...of</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/for-await...of",
+          "support": {
+            "chrome": {
+              "version_added": "63"
+            },
+            "chrome_android": {
+              "version_added": null
```

Comment:
This should match Chrome, as should `webview_android`.

Please wait to change this. I want to confirm the version number.

Reply:
Confirmed. Please change.